data/data_test.go

```diff
@@ -108,6 +148,101 @@ func TestGetAvailableVersions(t *testing.T) {
 	assert.Equal(t, versions, mockVersions, "component versions data")
 }
 
+func TestGetCluster(t *testing.T) {
+	mockCluster := getMockCluster(t)
+	cluster, err := GetCluster(mocks.InstalledMockData{}, &mocks.ClusterIDMockData{}, mocks.LatestMockData{})
+	assert.NoErr(t, err)
+	assert.Equal(t, cluster.ID, mockCluster.ID, "ID value")
+	for i, component := range cluster.Components {
+		// verify that GetCluster returned the expected mock Component data
+		assert.Equal(t, component.Component, mockCluster.Components[i].Component, "Component value")
+		// verify that GetCluster returned the expected mock Version data
+		assert.Equal(t, component.Version, mockCluster.Components[i].Version, "Version property")
+		// TODO
```

Comment:
added link to issue in https://github.com/deis/workflow-manager/pull/42/commits/edd945eb61ea39d16772e6c9f565e5486925827b